### PR TITLE
[Darkside] Updated Primitives CSS to use new token-prefix 

### PR DESCRIPTION
**Pull request by KenAJoh**
### Description

This update does not update the props for any of the primitives, as that can be a separate task

### CODE REVIEW

File: @navikt/core/react/src/layout/box/Box.tsx [hunk 1]
```diff
@@ -99,22 +100,28 @@ export const Box: OverridableComponent<BoxProps, HTMLDivElement> = forwardRef(
     },
     ref,
   ) => {
+    const themeContext = UNSAFE_useAkselTheme(false);
+    const prefix = themeContext ? "ax" : "a";
+
     const style: React.CSSProperties = {
       ..._style,
-      "--__ac-box-background": background
+      [`--__${prefix}-box-background`]: background
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/layout/box/Box.tsx]: You have used a/ax prefix several places that previously had "ac" prefix. This is just one example.

Comment (KenAJoh) [reply @navikt/core/react/src/layout/box/Box.tsx]: [Think i resolved most of it in this PR](https://github.com/navikt/aksel/pull/3423) 🤞 That merges into this PR

File: @navikt/core/react/src/layout/utilities/types.ts [hunk 2]
```diff
@@ -27,3 +27,17 @@ type FixedResponsiveT<T> = {
...
+/* focus */
+/* strong */
+/* subtle */
+/* info */
+/* danger */
+/* warning */
+/* success */
+
+/* Bg overlap */
+/* bg-default */
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/layout/utilities/types.ts]: Are these comments supposed to be there?

Comment (KenAJoh) [reply @navikt/core/react/src/layout/utilities/types.ts]: Removed them in the other PR to avoid merge-conflicts

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**